charts/argo-events/Chart.yaml

```diff
@@ -15,6 +15,7 @@ maintainers:
     url: https://argoproj.github.io/
 annotations:
   artifacthub.io/changes: |
+    - "[Fixed]: use policy/v1 for poddisruptionbudgets, when available
     - "[Fixed]: avoid app.kubernetes.io/version kubernetes label from exceeding maximum length (63)
```

Comment:
Please remove unrelated fixes from the annotation.

Reply:
Fixed